### PR TITLE
add stop method

### PR DESCRIPTION
**Pull request by minzcmu**
- add `stop` method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,21 @@ The callback is called with:
...
+### Stop
+The parameters required are:
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| config        | Object | Configuration Object |
+| config.buildId | String | The unique ID for a build to be stopped|
+| callback | Function | Callback `fn(err)` for when the build has been stopped |
+
+The `_stop` function will stop last build of the job with the `buildId` tag in jenkins.
```
Comment (FenrirUnbound) [inline README.md]: `stop`

**Comment by FenrirUnbound** [comment]
~~Since this is a backwards-compatible API change, you should bump the minor version.~~

JK, this is just implementing it

**Comment by d2lam** [comment]
LGTM 👍

**Comment by FenrirUnbound** [comment]
👍 

Looks fine to me

frgg9sp - imgur

**Pull request merged**